web/src/client/http.js

```diff
@@ -150,6 +150,27 @@ class HTTPClient {
     }
   }
 
+  /**
+   * @param {string} url - Endpoint URL (e.g., "/l10n/config").
+   * @param {object} data - Data to submit
+   * @return {Promise<object>} Server response.
+   */
+  async delete(url) {
+    const response = await fetch(`${this.baseUrl}/${url}`, {
+      method: "DELETE",
+      headers: {
+        "Content-Type": "application/json",
+      },
+    });
+
+    try {
+      return await response.json();
```

Comment:
The answer from the backend is empty, so we should the status code 204 (no content). In that case, there is not need to parse the response for a JSON body.